miniapp/brunel/plot_spikes_brunel.py

```diff
@@ -0,0 +1,62 @@
+import matplotlib.pyplot as plt
+from matplotlib import gridspec
+import random
+
+# reads two columns from file
+def read_file(file_name):
+    
+    ids = []
+    times = []
+    
+    file = open(file_name, 'r')
+    
+    for line in file:
+        ids += [float(line.split()[0])]
+        times += [float(line.split()[1])]
+    
+    file.close()
+
+    return (ids, times)
+
```

Comment:
How about making `nexc` and `ninh` command line parameters, so that this can be used to plot the output from arbitrary brunel simulations?